src/services/http/connector.js

```diff
@@ -0,0 +1,203 @@
+'use strict'
+
+module.exports = [
+  {
+    pin: 'role:auth,cmd:*',
+    prefix: 'auth',
+    map: {
+      /**
+         *
+         * @api {POST} /auth/login adminLogin
+         * @apiName adminLogin
+         * @apiGroup Auth
+         * @apiVersion  1.0.0
+         *
+         *
+         * @apiParam  {String} username admin username
+         *
+         * @apiParam  {String} password admin password
+         *
+         *
+         * @apiParamExample  {type} Request-Example:
+         * {
+         *     username : "justice"
+         *     password : "1234"
+         * }
+         *
+         *
+         * @apiSuccessExample {type} Success-Response:
+         *
+         * {
+         *       "data": {
+         *           "token": "eyJhbGciOiJIUzI1N............"
+         *       },
+         *       "status": 200
+         *
+         *
+         */
+      login: {
+        POST: true
+      },
+      /**
+         *
+         * @api {POST} /auth/signup adminSignup
```

Comment:
what else do you think need to enhance the `/auth/signup` API if it's a real product API? (You don't have to implement it, just a discussion)

Reply:
rate limiting to prevent brute attacks (either with express or allowing a proxy to handle it ), reduce the expiration time for the tokens, revoking tokens ,  handling sessions